db/models.py

```diff
@@ -1 +1,42 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=100)
+    description = models.TextField()
+
+    def __str__(self):
+        return self.name
+
+
+# Модель `Skill`, яка містить нове поле `race`
+class Skill(models.Model):
+    name = models.CharField(max_length=100)
+    description = models.TextField(null=True, blank=True)
+    bonus = models.IntegerField(default=0)
+
+    # Додаємо поле race як зовнішній ключ на модель Race
+    race = models.ForeignKey(Race, on_delete=models.CASCADE, default=1)  # Вказуємо значення за замовчуванням
+
+    def __str__(self):
+        return self.name
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(default="No description provided")
+
+    def __str__(self):
+        return self.name
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(blank=True, null=True)
+    bio = models.TextField(blank=True)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
```

Comment:
Consider adding a `related_name` attribute to the `race` ForeignKey in the `Player` model. This will help in reverse querying and improve code readability and maintainability.